pkg/handlers/v1/attribute_test.go

```diff
@@ -93,24 +93,24 @@ func TestHandle(t *testing.T) {
 			},
 			result: errors.New("oh noes"),
 		},
-		{
-			name: "attributed asset validation failure",
-			output: domain.NexposeAttributedAssetVulnerabilities{
-				NexposeAssetVulnerabilities: input,
-				BusinessContext: domain.CloudAssetDetails{
-					Hostnames: []string{"123"},
-				},
-			},
-			attributedErr:          nil,
-			attributeAndValidateOK: false,
-			AttributionFailureHandlerFunc: func(mockAttributionFailureHandler *MockAttributionFailureHandler) {
-				mockAttributionFailureHandler.EXPECT().HandleAttributionFailure(gomock.Any(), gomock.Any()).Return(nil)
-			},
-			AttributtedAssetValidationFunc: func(mockAssetValidator *MockAssetValidator) {
-				mockAssetValidator.EXPECT().Validate(gomock.Any(), gomock.Any()).Return(errors.New("validation error occurred here"))
-			},
-			result: errors.New("validation error occurred here"),
-		},
+		// {
```

Comment:
Task: We shouldn't commit commended code - it's confusing to the reader. If it needs to be commented, put the reason it's commented out in the comment with a jira ticket link or number so the reader can see why it's commented out and check what the current status of the ticket is.